w3/assignment3_2_2/TicTacToe/includes/game.h

```diff
@@ -0,0 +1,33 @@
+#ifndef GAME_H
+#define GAME_H
+
+#include <vector>
+#include <string>
+
+#define SIZE 3
```

Comment:
You should avoid using #define in C++ like this,  use for excample constexpr variables instead